External/FEXCore/Source/Interface/IR/IR.json

```diff
@@ -1109,31 +1100,46 @@
       ]
     },
 
-    "UCVTF": {
+    "FCVTFromGPR_U": {
+      "Desc": "Scalar op: Converts unsigned GPR to Scalar float",
       "HasDest": true,
       "SSAArgs": "1",
       "Args": [
         "uint8_t", "ElementSize"
       ]
     },
 
-    "SCVTF": {
+    "FCVTFromGPR_S": {
+      "Desc": "Scalar op: Converts signed GPR to Scalar float",
       "HasDest": true,
       "SSAArgs": "1",
       "Args": [
         "uint8_t", "ElementSize"
       ]
     },
 
-    "VUCVTF": {
+    "FCVTFloatToFloat": {
+      "Desc": "Scalar op: Converts float from one size to another",
+      "HasDest": true,
+      "SSAArgs": "1",
+      "Args": [
+        "uint8_t", "DstElementSize",
+        "uint8_t", "SrcElementSize"
+      ]
+    },
+
+    "VCVT_UToF": {
```

Comment:
VCVT is kind of opaque for a casual reader.

`Vector_UToF` would be infinity clearer and is only two chars longer. The `To` already implies conversion. 

Or perhaps Vector_UnsignedToFloat to go with the point above.